device_doctor/bin/main.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:args/args.dart';
+
+import 'package:device_doctor/src/config.dart';
+
+const String actionFlag = 'action';
+const String deviceOSFlag = 'deviceOS';
```

Comment:
nit: CLI args in the Flutter tooling follows the style of `--device-os`

Reply:
updated.